### ISSUE TITLE
`protolint:disable` doesn't turn off `FILE_NAMES_LOWER_SNAKE_CASE`

### ISSUE THREAD

**Issue by AlexCannonball** [yoheimuta/protolint#451]
Steps to reproduce:

- Create a file like `dot.separated.proto` with text

    ```proto
    // protolint:disable FILE_NAMES_LOWER_SNAKE_CASE
    syntax = "proto3";
    ````
- Run

    ```shell
    protolint lint -fix ./dot.separated.proto
    ```

Expected result: `dot.separated.proto` isn't renamed.

Factual result: the file is renamed to `dot_separated.proto`.

**Comment by yoheimuta** [comment]
@AlexCannonball Thank you for reporting.
I've fixed it. Please check out v0.55.4 to see it works.

**Comment by AlexCannonball** [comment]
Thank you, I've taken a quick look at v0.55.4, the issue is fixed.
